pages/search.js

```diff
@@ -267,47 +267,62 @@ class SearchPage extends React.Component {
 
 export { SearchPage as MockSearchPage };
 
-export const searchPageQuery = gql`
-  query SearchPage($term: String!, $types: [TypeOfCollective], $isHost: Boolean, $limit: Int, $offset: Int) {
-    search(term: $term, types: $types, isHost: $isHost, limit: $limit, offset: $offset, skipRecentAccounts: true) {
-      collectives {
+export const searchPageQuery = gqlV2/* GraphQL */ `
+  query SearchPage($term: String!, $type: [AccountType], $isHost: Boolean, $limit: Int, $offset: Int) {
+    accounts(
+      searchTerm: $term
+      type: $type
+      isHost: $isHost
+      limit: $limit
+      offset: $offset
+      skipRecentAccounts: true
+    ) {
+      nodes {
         id
         isActive
         type
         slug
-        path
         name
-        company
         imageUrl
-        backgroundImage
+        backgroundImageUrl
         description
         longDescription
         website
         currency
         stats {
           id
-          balance
-          totalAmountSpent
-          yearlyBudget
-          backers {
-            all
+          totalAmountSpent {
+            valueInCents
+          }
+          yearlyBudget {
+            valueInCents
           }
         }
-        parentCollective {
-          id
-          slug
+        ... on AccountWithParent {
+          parent {
+            id
+            slug
+          }
         }
-        memberOf(role: "BACKER") {
-          id
+        members(role: BACKER) {
+          totalCount
+        }
+        memberOf(role: BACKER) {
+          nodes {
+            id
+          }
```

Comment:
Doesn't seem you're doing anything with the nodes. Use `totalAcount` instead.

Reply:
You meant `totalCount`? I've changed that.